programs/ssl/ssl_client2.c

```diff
@@ -1325,11 +1393,16 @@ int main( int argc, char *argv[] )
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
     if( opt.key_opaque != 0 )
     {
-        /* coming soon: load key to key slot */
+        if( ( ret = wrap_key_as_opaque( &pkey, &key_slot ) ) != 0 )
```

Comment:
NB: That's a neat way to adapt the code with just a single line of change! :+1:

Reply:
Thanks :)